utils/merge.py

```
@@ -1,91 +1,174 @@
-    except Exception as e:
-        logger.error(f"Error occured while merging '{input_data}: {e}'")
+        result = subprocess.run(
+            cmd,
+            input=asin + "\n",
+            timeout=14400,
+            capture_output=True,
+            text=True,
+        )
+    except subprocess.TimeoutExpired:
+        message = f"m4b-merge timed out after 4 hours for ASIN: {asin}"
+        logger.error(message)
```

Ruff (0.15.0)

[warning] Use `logging.exception` instead of `logging.error`

Replace with `exception`

(TRY400)